spec/models/attendance_spec.rb

```diff
@@ -29,4 +29,16 @@
       expect(attendance.format_date).not_to eq(2.days.ago)
     end
   end
+
+  describe 'human_present' do
+    it 'human present' do
+      expect(attendance.human_present).to eq('Presente')
+    end
+
+    let!(:attendance_absent) { create(:attendance, present: false, ministry: ministry)}
+
+    it 'human absent' do
+      expect(attendance_absent.human_present).to eq('Ausente')
```

Comment:
No debería ser necesario ese `let!` definido en la línea 38.
Con que dentro de este `it` pongas `attendance.present = false` antes del `expect` debería ser suficiente